test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/ApexBaseTestClass.cs

```diff
@@ -41,9 +41,9 @@ public virtual NuGetApexTestService GetNuGetTestService()
 
         public virtual void Dispose()
         {
-            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("NUGET_TEST_CLOSE_VS_AFTER_EACH_TEST")))
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("NUGET_TEST_CLOSE_VS_AFTER_EACH_TEST")))
```

Comment:
The previous default was to not close VS after each test.  The new default is to close VS after each test.

If I set this environment variable, it will not close VS after each test.  That seems wrong.

I think you should just remove the `if` statement but keep the statement's body (minus the comment).

```C#
public virtual void Dispose()
{
    CloseVisualStudioHost();
}
```